t/cs.t

```diff
@@ -1872,8 +1872,8 @@ sub {
 --- request
 GET /c
 
---- error_log
-lua ssl certificate verify error: (18: self signed certificate)
+--- error_log eval
+qr/lua (ssl|tls) certificate verify error: \(18: self signed certificate\)/
```

Comment:
_note for the reader:_ 

upstream OpenResty seems to log:

> lua **ssl** certificate verify error [...]

...whereas the version I'm building w/ kong-build-tools logs:

> lua **tls** certificate verify error [...]

...I haven't been able to track down the source of this difference, so I changed the test to accept `ssl` or `tls` for my own convenience. It seems like a harmless change to me, but I'm happy to revert this if desired.

Reply:
Most likely because Kong has the `:tlshandshake` patch and uses it vs `:sslhandshake`